scripts/build_android.sh

```diff
@@ -39,9 +39,9 @@ if [ ! -d "$ANDROID_NDK" ]; then
   exit 1
 fi
 
-if [ -z "$PYTHON" ]; then
```

Comment:
Instead of putting ${CONDA_RUN} in all different place here, I think it would be nicer to use it when calling the script in the workflow instead.  For example

```
${CONDA_RUN} ./scripts/build_pytorch_android.sh x86
```

and (i wonder if the line below works, or if it's needed, but I guess you get the idea)

```
script: ${CONDA_RUN} ./android/run_tests.sh
```

Reply:
Yes, the latter is not needed, but I have implemented first recommendation.